tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/CreateOrAlterProcedureVisitor.cs

```diff
@@ -248,6 +255,42 @@ private MemberDeclarationSyntax AddPopulateCommandMethodForTableValuedParameters
                                     .WithNameColon(NameColon(ParameterNameForParameter(p)))).ToArray())));
     }
 
+    private MethodDeclarationSyntax[] AddGetOutputMethod(CreateOrAlterProcedureStatement node)
+    {
+        List<MethodDeclarationSyntax> result = new List<MethodDeclarationSyntax>();
+
+        foreach (ProcedureParameter parameter in node.Parameters)
+        {
+            if (parameter.Modifier != ParameterModifier.Output)
```

Comment:
nit: can include in the above `foreach` as a `Where` predicate: `node.Parameters.Where(p => p.Modifier == ParameterModifier.Output)`

Reply:
Thanks, updated.